assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -156,6 +156,9 @@ storage:
       mount:
         device: "/dev/disk/by-label/OEM"
         format: "ext4"
+        # This change has been added as a workaround the change introduced in ignition here:
+        # https://github.com/kinvolk/ignition/commit/1c735118f7091f6b22d10945bf3b5fdf9e50dbee.
```

Comment:
This is not really a workaround but the label needs to be here to match the OEM partition even if we fix the panic bug. Unfortunately the `format` is required because the original purpose of this `mount` directive is to create a filesystem and not reuse an existing filesystem. With the switch to btrfs for the OEM partition matching by `format` wouldn't work in your case here but with the label it works and we added a special case to allow format mismatches for the OEM partition.

Reply:
modified the commit as well as the comment.